_resources/port1.0/group/cmake-1.1.tcl

```diff
@@ -93,6 +93,8 @@ configure.args \
                     -DCMAKE_MODULE_PATH=${cmake_share_module_dir} \
                     -DCMAKE_FIND_FRAMEWORK=LAST \
                     -DCMAKE_EXPORT_COMPILE_COMMANDS=ON \
+                    -DCMAKE_C_COMPILER=${configure.cc} \
+                    -DCMAKE_CXX_COMPILER=${configure.cxx} \
                     -Wno-dev
```

Comment:
in cmake-1.1.tcl, CMAKE_C_COMPILER and CMAKE_CXX_COMPILER are set in a pre-configure block, just before `configure.cppflags` is parsed so its information is presented to CMake in a supported form (line 212 and onwards);

```tcl
    # force newer CMake versions to take a change in compiler choice into account
    # even if it is invoked in a build.dir that was configured before.
    if {${configure.cc} ne ""} {
        configure.args-append \
                    -DCMAKE_C_COMPILER=${configure.cc}
    }
    if {${configure.cxx} ne ""} {
        configure.args-append \
                    -DCMAKE_CXX_COMPILER=${configure.cxx}
    }
```

I'm inclined to think this is a bit more reliable approach than using `default configure.args`.

When using `default` I'd probably put these and related arguments in `configure.pre_args`.